app/views/admin/amendments/new.html.erb

```diff
@@ -150,6 +150,19 @@
         </div>
       <% end %>
     <% end %>
+
+    <% if @claim.decisions.any? %>
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-third">
+          <p class="govuk-body">Claim decision</p>
+        </div>
+        <div class="govuk-grid-column-two-thirds">
+          <p class="govuk-body">
+            <%= link_to "Amend decision", new_admin_claim_decision_undo_path(@claim, @claim.latest_decision), class: "govuk-link" %>
```

Comment:
I think we should show the current decision here as well, maybe before the link?

Reply:
Also, perhaps we should be explicit about the result of the action here? Maybe `Undo decision`?

Reply:
Yeah, nice. I've tweaked it to look like this now:

![image](https://user-images.githubusercontent.com/109774/77647843-100bb380-6f5f-11ea-9c6f-27aae5003e92.png)